piety-promoted-v3/modernized/10-chapter-10.adoc

```diff
@@ -226,14 +230,25 @@ added to your temporal ones, may not only perfect your happiness,
 but perpetuate it beyond the narrow limits of time, and qualify you, acceptably,
 to cast down your crown at the feet of Him who is King of kings and Lord of lords;
 who lives and reigns forever and ever.
-GEORGE AND SARAH DILLWYN, Of Burlington, New Jersey, North America.
+
+[.signed-section-signature]
+George and Sarah Dillwyn, Of Burlington, New Jersey, North America.
+
+[.signed-section-signature]
 SARAH GRUBB, Clonmel, Ireland.
+
+[.signed-section-signature]
 JOSHUA BEALE, Cork, Ireland
 
+[.signed-section-signature]
 Members of the religious Society of Friends in those countries and Great Britain,
 commonly called Quakers.
+
+[.signed-section-context-close]
 Mulheim on the Rhine, 29th of 9th Mo., called September, 1790.
 
```

Comment:
So this whole section is a mess...I wasn't sure what to do with both their names and where they are from, and what to label line 244. It seems that that was signifying they were Quakers, but I didn't know if it needed another "signed section signature" or what. 😟

Reply:
Yeah, that's a mess indeed. I actually tweaked what you did here a little (I added my own work to the PR, which I can do). But I struggled with it to the point that I just got it sort of looking OK, and I made a note to come back and work on it more later.  I kept most of what you did.